cloud_tasks/http_queues/create_http_queue_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import google.auth
+
+# HTTP Queues are currently in public beta
+from google.cloud import tasks_v2beta3 as tasks
+
+import create_http_queue
+
+
+def test_create() -> None:
+    # Use the default project and a random name for the test queue
+    _, project = google.auth.default()
+    name = uuid.uuid4().hex
```

Comment:
nit: can we add a prefix to all resources created during these tests so we can identify any leftoves in the future? For example by using `tests-tasks-v2beta3-` as a prefix.

Reply:
Okay, but I think just test-tasks- is enough for this purpose.